dask/dataframe/io/parquet/core.py

```diff
@@ -309,7 +313,7 @@ def to_parquet(
 
     # create parquet metadata, includes loading of existing stuff is appending
     meta, filenames = engine.create_metadata(df, fs, path, append=append,
-                                             partition_on=partition_on)
+             ignore_divisions=ignore_divisions, partition_on=partition_on)
```

Comment:
I think that `ignore_divisions` is maybe something that core.py should think about, rather than the engines.  Thoughts @martindurant ?